File: Helmut/1/solve.hs
import Data.List.Split (splitOn)
import Data.List (sort)

maxSum =  maximum . map (sum . map read) . splitOn [""] . lines
maxSum2=  sum . take 3 . reverse . sort . map (sum . map read) . splitOn [""] . lines

part1 = fmap   maxSum  $ readFile "input.txt" 

part2 = fmap maxSum2 $ readFile "input.txt"

File: Helmut/1/solve.rs
fn main() {
    let _ss = include_str!("input.txt");

    let mut calories = _ss
        .split("\n\n")
        .map(|cal| cal.lines().map(|x| x.parse::<i64>().unwrap()).sum())
        .collect::<Vec<i64>>();

    calories.sort();
    calories.reverse();

    println!("{}", calories.first().unwrap());
    println!("{}", calories.iter().take(3).sum::<i64>());
}
